doc/cookbook/source/examples/observers/least_angre_regression.rst

```diff
@@ -0,0 +1,50 @@
+=================================
+Observable Least Angle Regression
+=================================
+
+In this example, we will show how it is possible to obtain useful information from
```

Comment:
some grammar mistakes:
"In this example, we show how to retrieve information from a Shogun model during training"